### ISSUE TITLE
Dockerfile.centos "misses" FROM

### ISSUE THREAD

**Issue by lusitania** [docker/docker-bench-security#83]
If I use distros/Dockerfile.centos instead of the default one ``docker build`` yields this message:

```
Sending build context to Docker daemon 264.2 kB
Sending build context to Docker daemon
Step 0 : MAINTAINER smccarty@redhat.com
Please provide a source image with `from` prior to commit
```

Reordering 0: MAINTAINER <-> 1: FROM solves this issue.

Docker 1.7.1

Edit: I tried to create a pull request but I fail at the signing requirement, sorry

**Comment by konstruktoid** [comment]
Thanks for your contribution @lusitania! 
@fatherlinux could you update your `Dockerfile.centos` and send a PR with a reference to this issue?

**Comment by konstruktoid** [comment]
Closed by merging #84.
